docs/rules/jsx-handler-names.md

```diff
@@ -31,14 +31,16 @@ The following patterns are **not** considered warnings:
 "react/jsx-handler-names": [<enabled>, {
   "eventHandlerPrefix": <eventHandlerPrefix>,
   "eventHandlerPropPrefix": <eventHandlerPropPrefix>,
-  "checkLocalVariables": <boolean>
+  "checkLocalVariables": <boolean>,
+  "checkInlineFunction": <boolean>
 }]
 ...
 ```
 
 * `eventHandlerPrefix`: Prefix for component methods used as event handlers. Defaults to `handle`
 * `eventHandlerPropPrefix`: Prefix for props that are used as event handlers. Defaults to `on`
 * `checkLocalVariables`: Determines whether event handlers stored as local variables are checked. Defaults to `false`
+* `checkInlineFunction`: Determines whether event hanlders used as inline function. Defaults to `false`
```

Comment:
I think 'Determines whether event handlers set as inline functions are checked' would be better

Reply:
Oh thank you very much! That sentence is proper to understand. I'll reflect it next commit!